sretoolbox/container/image.py

```diff
@@ -473,19 +497,25 @@ def __eq__(self, other):
         if manifest_version == 1:
             layers_key = 'fsLayers'
         else:
-            manifest_media_type = manifest['mediaType']
-            other_manifest_media_type = other_manifest['mediaType']
+            manifest_content_type = self.content_type
+            other_manifest_content_type = other.content_type
 
-            if manifest_media_type != other_manifest_media_type:
+            if manifest_content_type != other_manifest_content_type:
                 return False
 
-            if manifest_media_type == MANIFEST_MEDIA_TYPE:
+            if manifest_content_type in [
+                SCHEMA2_MANIFEST_MEDIA_TYPE,
+                OCI_MANIFEST_MEDIA_TYPE
+            ]:
                 layers_key = 'layers'
-            elif manifest_media_type == MANIFEST_LIST_MEDIA_TYPE:
+            elif manifest_content_type in [
+                SCHEMA2_MANIFEST_LIST_MEDIA_TYPE,
+                OCI_IMAGE_INDEX_MEDIA_TYPE,
+            ]:
```

Comment:
same here... `elif manifest_content_type in MULTI_ARCH_MEDIA_TYPES`

Reply:
fixed in 60d2a89